package/com.unity.formats.usd/Editor/Scripts/Behaviors/UsdMenu.cs

```diff
@@ -74,7 +74,7 @@ public class UsdMenu : MonoBehaviour {
     [MenuItem("USD/Export Transform Overrides", priority = 50)]
     static public void MenuExportTransforms() {
       var root = Selection.activeGameObject.GetComponentInParent<UsdAsset>();
-      var overs = InitForSave(Path.GetFileNameWithoutExtension(root.usdFullPath) + "_overs.usda");
+      var overs = InitForSave(Path.GetFileNameWithoutExtension(root.usdFullPath) + "_overs", fileExtension:"usda");
```

Comment:
Does it have to be in `usda`? We can maybe suggest all compatible formats if you give: `"usd,usdc,usda"`. The save panel will then show as able to handle these formats. It would also make things easier if someone wants to overwrite an existing file.

Going further... I just saw what could be inconsistent:
`string path = EditorUtility.OpenFilePanel("Import USD File", "", "usd,usda,usdc,abc");`

`var overs = InitForSave(Path.GetFileNameWithoutExtension(root.usdFullPath) + "_overs", fileExtension:"usda");`

`public static Scene InitForSave(string defaultName, string fileExtension = "usd")`

` static void ExportSelected(BasisTransformation basisTransform,
                               string fileExtension = "usd",
                               bool exportMonoBehaviours = false)`

If I may, I would suggest to offer `usd,usda,usdc` by default for `InitForSave` and `ExportSelected()` and override it if there's a strong need of something different.

Reply:
I agree with Thomas, is there a particular reason we are limiting the export format to `usda`?

Reply:
You're right, there's no reason to force ascii over binary. Let's do as Thomas proposed. 
Ideally we should offer an option to select if the usd extension should be ascii or binary as both are valid. But let's keep that for when we refactor the exporter to expose export options.

Reply:
Just did it in 6f9f91f